src/views/catalog/CreateFromInstanceTypes/components/VMDetailsSection/components/VMDetailsBody/components/DetailsRightGrid.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+
+import { V1beta1DataSource } from '@kubevirt-ui/kubevirt-api/containerized-data-importer/models';
+import { V1alpha1PersistentVolumeClaim } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { t } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { getPVC } from '@kubevirt-utils/resources/template/hooks/useVmTemplateSource/utils';
+import { DescriptionList, GridItem } from '@patternfly/react-core';
+import VirtualMachineDescriptionItem from '@virtualmachines/details/tabs/details/components/VirtualMachineDescriptionItem/VirtualMachineDescriptionItem';
+
+import { humanizeBinaryBytes } from '../../../../../../../../utils/utils/humanize';
```

Comment:
```suggestion
import { humanizeBinaryBytes } from '@kubevirt-utils/utils/humanize';
```

Reply:
I had it that way, but got build errors. I'll try again.

Reply:
"Module not found: Error: Can't resolve '@kubevirt-utils/utils/humanize' in '/home/phillip/Development/kubevirt/kubevirt-plugin/src/views/catalog/CreateFromInstanceTypes/components/VMDetailsSection/components/VMDetailsBody/components"